app/delivery/send_to_providers.py

```diff
@@ -84,8 +84,11 @@ def send_sms_to_provider(notification):
                 notification.billable_units = template.fragment_count
                 update_notification_to_sending(notification, provider)
 
-        delta_milliseconds = (datetime.utcnow() - notification.created_at).total_seconds() * 1000
-        statsd_client.timing("sms.total-time", delta_milliseconds)
+        # Record StatsD stats to compute SLOs
+        statsd_client.timing_with_dates("sms.total-time", notification.sent_at, notification.created_at)
```

Comment:
this is also confusing to me, prototype of `timing_with_dates` is (stat, start, end, rate=1), shouldn't this be
`start = notification.created_at` and `end = time.now`

Reply:
Start and end are reversed on utils, but I promise that the first parameter should be in the newer than the second parameter to have a positive value.

https://github.com/cds-snc/notification-utils/blob/9d4336b7578c4a5b16fe9979015d3f6ac6d83aa4/notifications_utils/clients/statsd/statsd_client.py#L82-L85

Reply:
Happy to submit a PR there

Reply:
Submitted a utils PR https://github.com/cds-snc/notification-utils/pull/77